### DISCUSSION TITLE
sslocal在配置有插件的情况下，仍有可优化的地方

### DISCUSSION THREAD

**Discussion by dev4u** [shadowsocks/shadowsocks-rust#479] [General]
如题，如果在插件模式下，sslocal能用domain socket来接收上游（插件）的数据，这可以省去不必要的本地环境干扰。

**Comment by zonyitoo** [comment]
This requires support from plugins.

**Reply by dev4u** [reply]
嗯，你说得对。看来这事应该无解。这是先有鸡还是先有蛋的哲学问题。
